.github/pull_request_template.md

```diff
@@ -0,0 +1 @@
+gpt-updater:description
```

Comment:
I don't think this related to shortcut, do you mind to create a new PR for that?